### PR TITLE
Improve Go SDK documentation

### PR DESCRIPTION
**Pull request by psiemens**
## Description

Going to be talking about the Go SDK tomorrow, wanted to make sure the documentation is up to date.

### CODE REVIEW

**Review by 10thfloor — APPROVED** [review]

File: docs/content/sdks/golang/build-transaction.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ title: Build a Transaction
...
-Flow, like most blockchains, allows users to submit transactions that mutate
+Flow, like most blockchains, allows anybody to submit a transaction that mutates
 the shared global chain state. A transaction is an object that holds a payload,
 which describes the state mutation, and one or more authorizations that permit 
 the transaction to mutate the state owned by specific accounts.
 
 You can read more about transactions in the [transaction lifecycle documentation](/concepts/transaction-lifecycle).
 
-## Create the Transaction
+## Create the transaction
```
Comment (10thfloor) [inline docs/content/sdks/golang/build-transaction.md:13]: Erm, title case here should be "Create the Transaction"? Is this an AP rule?

Comment (psiemens) [reply docs/content/sdks/golang/build-transaction.md:13]: Ah yep I'll fix all the casing

**Review by vishalchangrani — APPROVED** [review]
lgtm

File: docs/content/sdks/golang/build-transaction.md (hunk 1 above)
Comment (10thfloor) [inline docs/content/sdks/golang/build-transaction.md:13]:
```suggestion
## Create the Transaction
```

File: docs/content/sdks/golang/build-transaction.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ func main() {
 }
 ```
 
-## Gas Limit
+## Gas limit
```
Comment (10thfloor) [inline docs/content/sdks/golang/build-transaction.md:101]:
```suggestion
## Gas Limit
```

File: docs/content/sdks/golang/build-transaction.md [hunk 3]
```diff
@@ -122,11 +122,11 @@ import (
 func main() {
   // ...
 
-  tx.SetGasLimit(1000)
+  tx.SetGasLimit(100)
 }
 ```
 
-## Reference Block
+## Reference block
```
Comment (10thfloor) [inline docs/content/sdks/golang/build-transaction.md:129]:
```suggestion
## Reference Block
```

File: docs/content/sdks/golang/build-transaction.md [hunk 4]
```diff
@@ -137,21 +137,214 @@ which takes about 10 minutes at average Mainnet block rates.
...
 
-- building transaction
-  - script
-  - arguments 
-  - gas 
-  - reference block (expiry)
-  - proposal
-  - payer
-  - authorizers
+## Proposal key
```
Comment (10thfloor) [inline docs/content/sdks/golang/build-transaction.md:166]:
```suggestion
## Proposal Key
```

File: docs/content/sdks/golang/sign-transaction.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ if err != nil {
 }
 ```
 
-## How Signatures Work on Flow
+## How signatures work on Flow
```
Comment (10thfloor) [inline docs/content/sdks/golang/sign-transaction.md:45]:
```suggestion
## How Signatures Work on Flow
```

**Review by 10thfloor — APPROVED** [review]

File: docs/content/sdks/golang/build-transaction.md (hunk 1 above)
Comment (10thfloor) [reply docs/content/sdks/golang/build-transaction.md]: I updated this.

**Pull request merged**